apps/backend/src/routes/login/+page.svelte

```diff
@@ -0,0 +1,161 @@
+<!-- JavaScript code -->
+<script lang="ts">
+  // import for forms
+  import { createForm } from 'svelte-forms-lib';
+
+  import { authStore } from '$lib/stores';
+  import type { FirebaseError } from 'firebase/app';
+
+  // import sign up function
+  import { signOut, signInEmailPassword } from '$lib/firebase/auth';
+
+  //import yup for form validation
+  import * as yup from 'yup';
+
+  let apiError: FirebaseError | null;
+
+  // Yup validation code
+  const { form, errors, handleChange, handleSubmit } = createForm({
+    initialValues: {
+      email: '',
+      password: '',
+    },
+    validationSchema: yup.object().shape({
+      email: yup.string().email().required('Please enter your Email.'),
+      password: yup.string().required('Please enter your Password.'),
+    }),
+
+    onSubmit: async (data) => {
+      console.log(data);
+      try {
+        await signInEmailPassword(data.email, data.password, (error) => (apiError = error));
+      } catch (error) {
+        console.error(error);
+      }
+    },
+  });
+</script>
+
+<!-- Armadillo Logo-->
+<div class="flex items-center justify-center p-10">
+  <img src="armadillo.png" alt="" />
+</div>
+
+<div class="flex items-center justify-center">
+  <!-- Login Form - Card -->
+  <div class="w-full max-w-lg p-10 border border-secondary rounded-md shadow-lg">
+    <!-- Login Form - Header -->
+    <p class="text-3xl font-semibold text-center mb-5">Welcome Back</p>
+
+    <!-- Login Form - Form Content -->
+    <form on:submit|preventDefault={handleSubmit}>
+      <!-- General Error Message -->
+      {#if apiError && (apiError.code === 'auth/user-not-found' || apiError.code === 'auth/wrong-password')}
+        <div class="p-4 flex flex-row bg-red-50 text-red-800 rounded-lg" role="alert">
+          <img src="errorlogo.svg" alt="" />
+          <div class="ml-2 text-sm font-medium">Invalid email or password, please try again</div>
+        </div>
+      {/if}
+
+      <!-- Form Content - Email Input -->
+      <div>
+        <label for="email" class="form-control w-full max-w-md">
+          <div class="label">
+            <span class="label-text">Email</span>
+          </div>
+          <input
+            id="email"
+            type="text"
+            name="email"
+            placeholder="Name@example.com"
+            on:change={handleChange}
+            bind:value={$form.email}
+            class="
+							input
+							input-bordered
+							w-full
+							max-w-md
+              focus:ring-secondary
+							focus:border-secondary
+							duration-300
+							focus:outline-none
+						" />
```

Comment:
When I say multiline, I don't mean this.

E.g.,
```
w-full max-w-md input input-bordered duration-300
focus:outline-none focus:ring-secondary focus:border-secondary
```

Reply:
I'm looking for the code to be compact yet readable

Reply:
Changed accordingly.